javascript-local-storage/data.js

```diff
@@ -0,0 +1,9 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJson = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJson);
```

Comment:
What if there's nothing there? 🤔 

![Screen Shot 2020-10-23 at 3 39 46 PM](https://user-images.githubusercontent.com/32880187/97059696-fc045080-1545-11eb-959d-bda3b6060db9.png)

Reply:
This doesn't look to have been addressed. Please read the direction I attached a screenshot of, there is a key step you are missing 👍